buildhub/settings.py

```diff
@@ -352,19 +352,10 @@ class Localdev(Base):
     @property
     def VERSION(self):
         fn = os.path.join(self.BASE_DIR, "version.json")
-        try:
-            with open(fn) as f:
-                return json.load(f)
-        except FileNotFoundError:
-            output = subprocess.check_output(
-                # Use the absolute path of 'git' here to avoid 'git'
-                # not being the git we expect in Docker.
-                ["/usr/bin/git", "describe", "--tags", "--always", "--abbrev=0"]
-            )  # nosec
-            if output:
-                return {"version": output.decode().strip()}
-            else:
-                return {}
```

Comment:
`git` isn't installed in the container, so if `version.json` isn't there, it fails. I thought about installing git, but I don't think we get a lot out of having the version when in local dev environment. So I simplified this down to what we do in Socorro.